### DISCUSSION TITLE
Add the issue number to the top toolbar

### DISCUSSION THREAD

**Discussion by muhguh** [gotson/komga#1543] [Ideas]
### Describe your suggested feature

When reading an issue it displays the Volume name and the issue name but not the number. For most comics the issue name isn't very relevant but it is nice to know what issue number you are reading. I think the format VolumeName  #IssueNumber - IssueName makes the most sense. 

e.g.
Current: Strange Tales (1951) - Nightmare!
Suggested: Strange Tales (1951) #164 - Nightmare!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
